.github/workflows/scan.yml

```
@@ -27,6 +27,9 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.16'
```

```suggestion
          go-version: '1.17.x'
```
To match https://github.com/fluxcd/image-reflector-controller/pull/190